azkaban_cli/azkaban.py

```diff
@@ -715,3 +716,30 @@ def fetch_executions_of_a_flow(self, project, flow, start, length):
         self.__catch_response_error(response, FetchExecutionsOfAFlowError)
 
         return response.json()
+
+    def pause_flow_execution(self, session, host, session_id, exec_id):
```

Comment:
I guess `session, host, session_id` are not needed here.

Reply:
Thanks for your feedback.
I'll remove it and make a new pull request.